src/app/shared/palette/money-gbp/money-gbp.builder.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { CaseField } from '../../domain/definition/case-field.model';
+
+@Injectable()
+export class MoneyGBPCaseFieldBuilder {
```

Comment:
is palette the correct package for this?

Reply:
Following the below comment I have extracted a new MoneyGBPCaseField class so not relevant anymore.